### DISCUSSION TITLE
Help with testing basic ios app on iphone

### DISCUSSION THREAD

**Discussion by chrisjchrisj** [codemagic-ci-cd/codemagic-docs#2227] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

A developer provided me with some code(folders) for a basic ios app. I've downloaded it onto my iPhone. I'm told I need to connect my iphone to a Mac (which I don't have - only pc) to see that it works successfully.
I have the files on my iphone. How do I test it? Can I use Codemagic to accomplish this?
(I don't have "software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version).", but I checked "I confirm", also confirmed "I confirm I am able to do a fresh clone of the repository and build the project on my local computer") in order to post here.

### Expected results

I'd like some help

### Actual results

I look forward to help

**Comment by himesh-cm** [comment]
Hi @chrisjchrisj, you can deploy to Testflight and access the app via Testflight on your iPhone. [Here](https://docs.codemagic.io/yaml-quick-start/building-a-native-ios-app/) is our quick start guide.

**Comment by himesh-cm** [comment]
Hi @chrisjchrisj,  I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message.
